apidoc/View.yml

```diff
@@ -917,7 +917,7 @@ properties:
     since: "3.2.3"
     platforms: [android]
 
-  - name: zoom
+  - name: zoomLevel
```

Comment:
@garymathews : could you kindly confirm, if the "since" attribute for this zoomLevel property also needs to be updated? If yes, then how to determine the value?

Reply:
Since it was impossible for devs to read the "zoom" property before, then yes, change the "since" to the next version of this module.

Reply:
the suggested change is done and this PR updated for same.